### PR TITLE
Service Interconnect zstream 4.16 and firewatch updates

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.15-lp-interop-rhsi-interop-ibmcloud periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.16-lp-interop-rhsi-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.15-lp-interop-rhsi-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.15-lp-interop-rhsi-interop-ibmcloud periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.16-lp-interop-rhsi-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.15-lp-interop-rhsi-interop-ibmcloud

File: ci-operator/config/skupperproject/skupper-openshift-smoke-test-image/skupperproject-skupper-openshift-smoke-test-image-main__service-interconnect-ocp4.14-lp-interop.yaml [hunk 1]
```diff
@@ -64,21 +59,13 @@ tests:
...
-            ],
-          "success_rules":
-            [
-              {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
             ]
         }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/ibm-ipi/lp-interop.json
       FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.14-lp","self-managed-lp","rhsi-lp"]'
       FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
-      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
```
Comment (Simranpal) [inline ci-operator/config/skupperproject/skupper-openshift-smoke-test-image/skupperproject-skupper-openshift-smoke-test-image-main__service-interconnect-ocp4.14-lp-interop.yaml:81]: This was changed for only 4.14?

Comment (smatula) [reply ci-operator/config/skupperproject/skupper-openshift-smoke-test-image/skupperproject-skupper-openshift-smoke-test-image-main__service-interconnect-ocp4.14-lp-interop.yaml:81]: No, It is removed from 4.15 and 4.16 for ibm also... It is the default for the ibmcloud workflow and does not need to be set to true in config.

Comment (smatula) [reply ci-operator/config/skupperproject/skupper-openshift-smoke-test-image/skupperproject-skupper-openshift-smoke-test-image-main__service-interconnect-ocp4.14-lp-interop.yaml:81]: Still needs to be done for aws workflow.

**Pull request merged**